src/test/java/com/exasol/errorcodecrawlermavenplugin/ErrorCodeCrawlerMojoIT.java

```diff
@@ -272,28 +272,8 @@ ITVerifier withSubProject() throws IOException {
             rootModel.addModule("sub-project");
             rootModel.setPackaging("pom");
 
-            final Path parentPomPath = this.projectDir.resolve("parent-pom");
-            final TestMavenModel parentModel = mavenModel(CURRENT_VERSION, null, null);
-            parentModel.setArtifactId("parent-pom");
-            parentModel.setPackaging("pom");
-            Files.createDirectories(parentPomPath);
-            final InputStream stream = ErrorCodeCrawlerMojoIT.class.getClassLoader().getResourceAsStream("testProject/" + CONFIG_NAME);
-            Files.copy(Objects.requireNonNull(stream), //
-                    parentPomPath.resolve(CONFIG_NAME), //
-                    StandardCopyOption.REPLACE_EXISTING);
-
-            parentModel.writeAsPomToProject(parentPomPath);
-
-            Parent parent = new Parent();
-            parent.setVersion(parentModel.getVersion());
-            parent.setGroupId(parentModel.getGroupId());
-            parent.setArtifactId(parentModel.getArtifactId());
-            parent.setRelativePath("../parent-pom/pom.xml");
-
```

Comment:
The previous logic was incorrect: I assumed that if project has parent, then it is considered sub-module and should have sub-project name as prefix in the source path. Unfortunately, this assumption was incorrect. The actual sub-module has nothing to do with parent pom module. @kaklakariada  suggested to use use **executionRootDirectory** based on property **session.executionRootDirectory**, which finally solved the issue.

In order to avoid confusion, I have removed parent pom from integration test with sub-project, because with the fixed logic parent pom has nothing to do with module - sub-module relation.

But after discussion with @kaklakariada , it was suggested to add separate integration test, which would reproduce the issue, namely: project with parent pom should have correct source path, without project name as prefix. 
So I moved this code to the new separate integration test.